app/src/main/java/com/kickstarter/ui/activities/NewsletterActivity.kt

```diff
@@ -47,23 +47,40 @@ class NewsletterActivity : BaseActivity<NewsletterViewModel.ViewModel>() {
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(this::showOptInPrompt)
 
+        alumni_switch.setOnClickListener { viewModel.inputs.sendAlumniNewsletter(alumni_switch.isChecked) }
+        arts_news_switch.setOnClickListener { viewModel.inputs.sendArtsNewsNewsletter(arts_news_switch.isChecked) }
+        films_switch.setOnClickListener { viewModel.inputs.sendFilmsNewsletter(films_switch.isChecked) }
+        games_we_love_switch.setOnClickListener { viewModel.inputs.sendGamesNewsletter(games_we_love_switch.isChecked) }
         happening_switch.setOnClickListener { viewModel.inputs.sendHappeningNewsletter(happening_switch.isChecked) }
-        news_events_switch.setOnClickListener { viewModel.inputs.sendPromoNewsletter(news_events_switch.isChecked) }
```

Comment:
I had a question about this. I think we want to add it back?

Reply:
Danny said it we're no longer showing that Newsletter. @dannyalright  Can you confirm please?

Reply:
`sendPromoNewsletter`? Which newsletter is that?

Reply:
@dannyalright The `News & Events` Newsletter

Reply:
gotcha. I meant for that to be re-added. Sorry for the confusion

Reply:
Gotcha no problem I'll add it back